woocommerce/woocommerce-android
# Code scanner UI in the order detail screen

Closes: #8941 


### Description

This PR adds an entry point to the "Scanned and Deliver" project in the Order details screen. It basically adds a barcode icon right next to the "Add Products" button in the Order details screen.

More context: pecCkj-GT-p2

<img width="626" alt="Screenshot 2023-05-17 at 1 01 07 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/4029fc9a-be85-4a61-8c4d-5fc8f8672451">


### Testing instructions

1. Navigate to the order creation screen (orders -> "+")
2. Ensure you see the barcode icon next to the "Add products" button in the `Products` section
3. Tap on the barcode icon
4. Ensure the scanning screen opens

### Images/gif

<img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/8bb766fd-4e32-4fe5-a5ad-4a5be5d5caef"/>

<img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/9df958d7-05a0-4c48-be66-09f7ee2899e4"/>
